spec/index.md

```diff
@@ -5064,9 +5161,11 @@ S.messages = Older_messages · ResponseMessage RM · Younger_messages
 RM.origin = FromCanister {
     call_context = Ctxt_id
     callback = Callback
+    deadline = D
```

Comment:
TODO: Add a condition here (and the transition below) to say that `ctxt_id \in dom(S.call_contexts)`; and also remove the invariant that we don't have "holes" in our call context graph